plugins/terraform-cloud-processor-backend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@backstage/plugin-terraform-cloud-processor-backend",
```

Comment:
I'm thinking this perhaps is better off as a catalog module, similar to the LDAP and MSGraph ones. So it's be something like `plugin-catalog-backend-module-terraform-cloud`

Reply:
makes sense - will update!

Reply:
Thanks! seems we have a duplication going on now that needs to be cleaned up :grin: